src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -66,11 +68,11 @@ var DOMRenderer = ReactFiberReconciler({
     return true;
   },
 
-  commitUpdate(domElement : Instance, oldProps : Props, newProps : Props, children : HostChildren<Instance>) : void {
-    domElement.innerHTML = '';
-    recursivelyAppendChildren(domElement, children);
```

Comment:
Can you avoid this change?
It's a bit messy because the histories diverged.

I only changed this method here because Flow complained at `domElement.textContent = newProps.children;` assignment. However in `15-stable` this code didn't exist in the first place so we can safely leave the old implementation here.